### PR TITLE
SPUI-228 > Add action menu and restructure folders

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: lib/app/screens/account/view/account.dart [hunk 1]
```diff
@@ -9,13 +9,13 @@ import '../../../../shared/providers/service_providers.dart';
...
-  const Profile();
+class Account extends HookWidget {
+  const Account();
 
   @override
   Widget build(BuildContext context) {
-    final state = useProvider(logoutNotipod);
-    final notifier = useProvider(logoutNotipod.notifier);
+    final logout = useProvider(logoutNotipod);
+    final logoutN = useProvider(logoutNotipod.notifier);
```
Comment (VovaMedvediev) [inline lib/app/screens/account/view/account.dart:18]: new convention for notifier variables?

File: lib/app/screens/navigation/view/components/bottom_navigation_menu/bottom_navigation_item.dart [hunk 2]
```diff
@@ -19,27 +21,43 @@ class BottomNavigationItem extends HookWidget {
...
+    final intl = useProvider(intlPod);
     final navigation = useProvider(navigationStpod);
     final color = navigation.state == index ? Colors.black : Colors.grey;
 
     return Expanded(
       child: InkResponse(
         radius: 50.r,
-        onTap: () => navigation.state = index,
+        onTap: () {
+          if (index == 2) {
```
Comment (VovaMedvediev) [inline lib/app/screens/navigation/view/components/bottom_navigation_menu/bottom_navigation_item.dart:32]: add variable for "2" index please

**Review by VovaMedvediev — CHANGES_REQUESTED** [review]

File: lib/app/screens/account/view/account.dart (hunk 1 above)
Comment (eli1stark) [reply lib/app/screens/account/view/account.dart:18]: yes

File: lib/app/screens/navigation/view/components/bottom_navigation_menu/bottom_navigation_item.dart (hunk 2 above)
Comment (eli1stark) [reply lib/app/screens/navigation/view/components/bottom_navigation_menu/bottom_navigation_item.dart:32]: I will create private variable

**Pull request merged**